big_tests/tests/mod_http_notification_SUITE.erl

```diff
@@ -66,68 +56,57 @@ end_per_suite(Config) ->
     escalus:delete_users(Config, escalus:get_users([alice, bob])),
     escalus:end_per_suite(Config).
 
-init_per_group(mod_http_notification_tests, Config) ->
-    set_worker(Config);
-init_per_group(mod_http_notification_tests_with_prefix, Config) ->
-    set_worker(Config, "/prefix").
+init_per_group(no_prefix, Config) ->
+    set_modules(Config, []);
+init_per_group(with_prefix, Config) ->
+    set_modules(Config, [{path, "/prefix"}]).
 
 end_per_group(_GroupName, Config) ->
-    ejabberd_node_utils:call_fun(mongoose_wpool, stop, [http, global, http_pool]),
     dynamic_modules:restore_modules(host(), Config),
     ok.
 
 init_per_testcase(CaseName, Config) ->
+    create_events_collection(),
     start_http_listener(CaseName, get_prefix(Config)),
+    start_pool(),
     escalus:init_per_testcase(CaseName, Config).
 
 end_per_testcase(CaseName, Config) ->
-    http_helper:stop(),
+    stop_pool(),
+    stop_http_listener(CaseName),
+    clear_events_collection(),
     escalus:end_per_testcase(CaseName, Config).
 
-start_http_listener(simple_message, Prefix) ->
-    Pid = self(),
-    http_helper:start(http_notifications_port(), Prefix, fun(Req) -> process_notification(Req, Pid) end);
-start_http_listener(simple_message_no_listener, _) ->
-    ok;
-start_http_listener(simple_message_failing_listener, Prefix) ->
-    http_helper:start(http_notifications_port(), Prefix, fun(Req) -> Req end);
-start_http_listener(proper_http_message_encode_decode, Prefix) ->
-    Pid = self(),
-    http_helper:start(http_notifications_port(), Prefix, fun(Req) -> process_notification(Req, Pid) end).
-
-process_notification(Req, Pid) ->
-    {ok, Body, Req1} = cowboy_req:read_body(Req),
-    Req2 = cowboy_req:reply(200, #{<<"content-type">> => <<"text/plain">>}, <<"OK">>, Req1),
-    Pid ! {got_http_request, Body},
-    Req2.
-
 %%%===================================================================
 %%% offline tests
 %%%===================================================================
-proper_http_message_encode_decode(Config) ->
-    Receiver= <<"bob">>,
+proper_http_message_encode_decode(Config0) ->
+    Config = escalus_fresh:create_users(Config0, [{alice, 1}, {bob, 1}]),
```

Comment:
Why is this changed? What prevents us from using `escalus_story`?

Reply:
Users are fresh now, containing the test name, and for some mysterious reason I'm not aware of, `escalus_story` was taking several seconds to run, while as it is now takes less than 0.1s

Reply:
Was it also the case with `fresh_story`?

Reply:
Don't remember trying with fresh_story, but I probably chose not to because in `do_simple_message` we were disconnecting users independently. I'll try out quickly with fresh_story nevertheless, might make it cleaner 🤔